src/commands/quickstart.ts

```diff
@@ -196,20 +199,21 @@ export default class Quickstart extends Command {
       logCb?: LogCallback;
       warnCb?: LogCallback;
     }
-  ): Promise<string | undefined> {
+  ): Promise<void> {
     const response: { value: string } = await inquirer.prompt({
       name: 'value',
       message: `Enter ${name} of ${authType} security for "${provider}" This value will be stored locally in .env file.`,
       type: 'input',
     });
-    if (process.env[envVariableName]) {
+    const variableName = envVariableName.startsWith('$')
+      ? envVariableName.substring(1)
+      : envVariableName;
+    if (this.envContent.includes(variableName)) {
```

Comment:
The order of operations is still wrong. We shouldn't prompt for a value when we can then discard it if the variable is set. The `this.envContent.includes(variableName)` check should come before the `prompt`.

Reply:
And about the check what if one of the keys is called "FOO" and the other is "FOO2". Now if "FOO2" is already set we skip "FOO" as well because that substring already appears in the file.

Since we know the .env file structure we should look for "FOO=" or use regex and look for `^FOO=`